core/src/test/java/org/elasticsearch/index/query/MoreLikeThisQueryBuilderTests.java

```diff
@@ -204,6 +207,16 @@ protected MoreLikeThisQueryBuilder doCreateTestQueryBuilder() {
         return queryBuilder;
     }
 
+    /**
+     * we don't want to shuffle the "doc" field internally in {@link #testFromXContent()} because even though the
+     * documents would be functionally the same, their {@link BytesReference} representation isn't and thats what we
+     * compare when check for equality of the original and the shuffled builder
+     */
+    @Override
+    protected String[] shuffleProtectedFields() {
+        return SHUFFLE_PROTECTED_FIELDS;
```

Comment:
Had to make this change now that `XContent` is shuffled for objects inside lists.